config/api.yml

```diff
@@ -758,10 +758,26 @@
         :identifier: cloud_tenant_tag
   :cloud_volume_snapshots:
     :description: Cloud Volume Snapshots
+    :identifier: cloud_volume_snapshot
     :options:
-    - :subcollection
-    :verbs: *gpppd
```

Comment:
Openstack supports updating cloud_volume_snapshots, why delete PUT/PATCH ?

Reply:
Currently, Autosde doesn't support updating snapshots, but I think I can leave it as it was, I'll check it